.travis.yml

```diff
@@ -28,6 +28,10 @@ before_script:
 
 script:
   - cargo test
+  - cargo test --features rdcore
   - bash -c 'if [[ "$TRAVIS_RUST_VERSION" == "$CLIPPY_RUST_VERSION" ]]; then
       cargo clippy -- -D warnings;
     fi'
+  - bash -c 'if [[ "$TRAVIS_RUST_VERSION" == "$CLIPPY_RUST_VERSION" ]]; then
+      cargo clippy --features rdcore -- -D warnings;
```

Comment:
Minor: can't we fold this into the previous script line?

Reply:
Sure, if you'd prefer.  Done.